calculator_test.go

```diff
@@ -68,15 +70,16 @@ func TestSqrt(t *testing.T) {
 	}{
 		{
 			a:    2,
-			want: 4,
+			want: math.Sqrt(2),
```

Comment:
This is a bit dangerous, since we're also using `math.Sqrt` to implement the function! Of course, we hope that `math.Sqrt` works, but even if it does, we'd be better off avoiding it. We should always work out our test expectations _beforehand_ wherever we can (and here, we can).